CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.21.0
 
 * Add support for `secretPrefix` property for User Operator to prefix all secret names created from KafkaUser resource.
+* Allow configuring labels and annotations for Cluster CA public key secrets
```

Comment:
should we say "certificate" instead of "public key"? In the end it's a public key ... self-signed ... so a self-signed certificate but we use more the "certificate" term in the doc than public key.

Reply:
I think public key is good to make it clear it is only the public key secret and not the private key secret. When you use certificate it is not clear which one is meant (and that only one is supported.)

Reply:
to be honest saying "certificate" should be clear enough that it's the public part of the encryption-related stuff. You distribute certificates, you don't distribute just public keys (and of course not private ones).

Reply:
I don't think it is clear. But whatever.